sonar-plugin-api/src/main/java/org/sonar/api/measures/Measure.java

```diff
@@ -666,14 +668,12 @@ public boolean equals(Object o) {
     if (metricKey != null ? !metricKey.equals(measure.metricKey) : (measure.metricKey != null)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1126)